admin/src/js/components/matches/index.js

```diff
@@ -114,6 +115,20 @@ class Match extends React.Component {
         })
     }
 
+    nagHost(e) {
+        if (e) {
+            e.preventDefault();
+        }
+
+        return fetch(`/api/match/${this.props.match.id}/email/host_nag`, {
+            method: 'POST',
+            credentials: 'include',
+            headers: {
+                'Content-Type': 'application/json'
+            }
+        }).then(d => console.log("Mailed"));
```

Comment:
Since this is an action which leads to sending out email to our users, I think we should give the admin feedback in the UI